rustls-mbedcrypto-provider/Cargo.toml

```diff
@@ -25,7 +25,9 @@ bit-vec = "0.6.3"
 
 
 [dev-dependencies]
-rustls = { version = "0.22.1", default-features = false, features = ["ring"] }
+rustls = { git = "https://github.com/fortanix/rustls", branch = "ffdhe", default-features = false, features = [
```

Comment:
Will this `ffdhe` branch becomes the final branch we will use before upstream?
If so, I would suggest to use `tag` instead, so `cargo` could automatically update dependencies once we update the branch and use new `tag`. Otherwise, we need to manually run `cargo update -p rustls` to update the `Cargo.lock` files.

Reply:
`rev` also is an option, but it will lack some context.

Reply:
We could do this after the `ffdhe` feature is fully done.